samples/snippets/src/main/java/com/example/spanner/admin/generated/GetInstanceConfigSample.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.admin.generated;
+
+//[START spanner_get_instance_config]
+
+import com.google.cloud.spanner.admin.instance.v1.InstanceAdminClient;
+import com.google.spanner.admin.instance.v1.InstanceConfig;
+import java.io.IOException;
+
+public class GetInstanceConfigSample {
+
+  static void getInstanceConfig() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    final String instanceConfigName = "nam6";
+    getInstanceConfig(instanceConfigName);
+  }
+
+  static void getInstanceConfig(String instanceConfigName) throws IOException {
+    final InstanceAdminClient instanceAdminClient = InstanceAdminClient.create();
+
+    final InstanceConfig instanceConfig = instanceAdminClient.getInstanceConfig(instanceConfigName);
```

Comment:
I don't think this method would work with the example value that we have in our sample (`nam6`), as that is not a fully qualified name.

Reply:
Your'e right. The integration test was using the fully qualified name. I have now made this consistent.